eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -65,9 +65,12 @@ function Get-TocMapping {
     $orderServiceMapping = @{}
 
     foreach ($artifact in $artifacts) {
-        $packageInfo = $metadata | ? { $_.Package -eq $artifact -and $_.Hide -ne "true" }
+        $packageInfo = $metadata | ? { $_.Package -eq $artifact }
         $serviceName = ""
         $displayName = ""
+        if ($packageInfo.Hide -eq "true") {
+            continue
+        }
         if (!$packageInfo) {
```

Comment:
I suggest we just continue if we don't have a packageInfo entry yet, instead of putting it into the Other section.

Reply:
So the package in blob storage but not in csv file will be hidden.

Reply:
Yes I think that is OK for now. They will end up in the csv file after they fully ship.

Reply:
I will leave the logging message for information purpose.